docs/notebooks/simulation/particle_stepper.ipynb

```diff
@@ -4,10 +4,7 @@
    "cell_type": "code",
```

Comment:
<p>"ParticleTracker takes arrays of particle positions and velocities of the shape [nparticles, 3], so these arrays represent one particle starting at the origin." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>